src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Ensure that each BEM block has its own separate file as per the task requirements. This helps in maintaining modularity and reusability of components.